src/Longitude.cpp

```diff
@@ -2,6 +2,17 @@
 #include "Utility.hpp"
 #include "Longitude.hpp"
 
+// define INDI in the local configuration if you want it to work
+#ifndef INDI
+    #define INDI 0
```

Comment:
I would rename this `#define` to be a little more specific. `INDI_LONGITUDE_HACK` maybe?

Reply:
Actually... it's the opposite, at least from my understanding. What it does is preserve the current parsing on Gg12 -> 180 - 12 for users of the current ASCOM driver, when the correct value should be - 12. The parsing assumes that KStars/Meade/ASCOM will have a 180 - x offset added when in fact it does not.

The whole thing is explained here (and I should referenced this in the subject): https://github.com/OpenAstroTech/OpenAstroTracker-Firmware/issues/166

I just named it INDI (which is a lame name, agreed, maybe INDI_SUPPORT) because I have a feeling there might be more stuff coming in to support KStars and I didn't want flags for all of them.

Reply:
We can re-evaluate at a later date if more INDI-specific changes are needed. More specific is better in the current case.